physics/cires_orowam2017.F90

```diff
@@ -0,0 +1,354 @@
+module cires_orowam2017
```

Comment:
This file should be renamed so it differs from cires_orowam2017.f by more than just the fortran format extension (f vs. F90)